downloader/downloader.go

```diff
@@ -587,6 +587,13 @@ func (downloader *Downloader) Download(data *extractors.Data) error {
 				stream = data.Streams[s.ID]
 				break
 			}
+			for _, part := range s.Parts {
+				if part.Ext == "m4a" {
+					isFound = true
+					stream = data.Streams[s.ID]
+					break
+				}
+			}
```

Comment:
What's this for?

Reply:
bilibili has not  a separate audio stream, but usually mp4 + m4a, so only download the m4a part?